spec/requests/api/picture_spec.rb

```diff
@@ -61,4 +61,44 @@ def expect_result_to_include_picture_href(source_id)
       expect_result_to_include_picture_href(service_request.id)
     end
   end
+
+  describe 'POST /api/pictures' do
+    it 'rejects create without an appropriate role' do
+      api_basic_authorize
+
+      run_post pictures_url, :extension => 'png', :content => 'content'
+
+      expect(response).to have_http_status(:forbidden)
+    end
+
+    it 'creates a new picture' do
+      api_basic_authorize collection_action_identifier(:pictures, :create)
+
+      expect do
+        run_post pictures_url, :extension => 'png', :content => 'content'
```

Comment:
Since this also serves as an example, WDYT about making content semi-realistic, e.g. represent a single pixel picture?